src/components/Comments/WriteComment.js

```diff
@@ -123,7 +122,7 @@ const WriteComment = ({
           testID="changeinput"
           ref={commentInputRef}
           placeholder={StringConstant.commentBoxDefaultPlaceholder}
-          placeholderTextColor={colors.gray}
+          placeholderTextColor={COLORS.gray8}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The placeholder text color has been updated. Ensure that `COLORS.gray8` provides sufficient contrast against the background for readability.
```diff
- placeholderTextColor={colors.gray}
+ placeholderTextColor={COLORS.gray8}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->